src/jtag/drivers/remote_bitbang.c

```diff
@@ -188,16 +189,14 @@ static int remote_bitbang_sample(void)
 static bb_value_t remote_bitbang_read_sample(void)
 {
 	if (remote_bitbang_recv_buf_start == remote_bitbang_recv_buf_end) {
```

Comment:
This could be put to a separate function `remote_bitbang_recv_buf_is_empty()` for readability.